starfish/image/_filter/clip.py

```diff
@@ -59,8 +59,12 @@ def _clip(image: Union[xr.DataArray, np.ndarray], p_min: int, p_max: int) -> np.
         return image.clip(min=v_min, max=v_max)
 
     def run(
-            self, stack: ImageStack, in_place: bool=False, verbose: bool=False,
-            n_processes: Optional[int]=None
+            self,
+            stack: ImageStack,
+            in_place: bool=False,
+            verbose: bool=False,
+            n_processes: Optional[int]=None,
+            *args,
```

Comment:
where would we actually use these extra *args? Should we pass them into image.clip as well?